requirements.txt

```diff
@@ -1,6 +1,7 @@
 # Dev tools:
 black==24.2.0
-pre-commit==3.5.0
+pre-commit==3.5.0; python-version < 3.9
+pre-commit==3.6.2
```

Comment:
I think you need `; python_version >= 3.9` at the end of this line, otherwise Python 3.8 will get confused as to which to use

Reply:
good point, fixed